server/app/app.go

```diff
@@ -15,15 +16,24 @@ type UserSurveyApp struct {
 	apiClient  *pluginapi.Client
 	botID      string
 	debugBuild bool
+	manifest   *mmModel.Manifest
 }
 
-func New(api plugin.API, store store.Store, getConfigFunc func() *model.Config, driver plugin.Driver, debugBuild bool) (*UserSurveyApp, error) {
+func New(
+	api plugin.API,
+	store store.Store,
+	getConfigFunc func() *model.Config,
+	driver plugin.Driver,
+	debugBuild bool,
+	manifest *mmModel.Manifest,
+) (*UserSurveyApp, error) {
 	app := &UserSurveyApp{
 		api:        api,
 		store:      store,
 		getConfig:  getConfigFunc,
 		apiClient:  pluginapi.NewClient(api, driver),
 		debugBuild: debugBuild,
+		manifest:   manifest,
```

Comment:
I fail to see where the manifest is used in all this. Why is it needed?

Reply:
Nice catch! I was using it before, but then wasn't needed. I've removed it now.